projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/graphql/model/V2DataJobDeployment.java

```diff
@@ -28,4 +28,6 @@
    private DataJobExecution.StatusEnum lastExecutionStatus;
    private OffsetDateTime lastExecutionTime;
    private Integer lastExecutionDuration;
+   private Integer successfulDataJobExecutions;
```

Comment:
```suggestion
   private Integer successfulExecutions/executionsSucceeded;
```

What do you think which one is better? (The `DataJob` in the property name is redundant since the element is part of the job itself.) I am in favor of `executionsSucceeded`. @tpalashki @mrMoZ1

Reply:
First option is more descriptive, second is a bit shorter. To me it doesn't really matter which one we pick.

Reply:
The first option is more associative with a count. However, in Kubernetes jobs, the number of successful pods is named `succeeded` and is a number, so the second option makes sense as well.

Reply:
:)

Reply:
@mrMoZ1 it's up to you.